src/status_im2/common/bottom_sheet/styles.cljs

```diff
@@ -22,14 +22,25 @@
    :top              0
    :background-color colors/neutral-100})
 
+(defn container
+  []
+  {:position :absolute
+   :left     0
+   :right    0
+   :top      0
+   :bottom   0
+   :overflow :hidden})
+
+(def selected-background
```

Comment:
should this be a function (defn)?

Reply:
:+1: It must be, otherwise switching theme doesn't update the map.